Sources/Position.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  PositionVector.swift
+//  Euclid
+//
+//  Created by Ioannis Kaliakatsos on 29.11.2020.
+//  Copyright © 2020 Nick Lockwood. All rights reserved.
+//
+
+import Foundation
+
+public struct Position: CartesianComponentsRepresentable {
+    public let x: Double
+    public let y: Double
+    public let z: Double
+
+    public init(x: Double = 0, y: Double = 0, z: Double = 0) {
+        self.x = x
+        self.y = y
+        self.z = z
+    }
+}
+
+public extension Position {
+    static let origin = Position()
+}
+
+public extension Position {
```

Comment:
Can't a position be rotated?

Reply:
True, somehow this went through the cracks. Will fix